src/injected/focus-change-handler.ts

```diff
@@ -34,4 +36,29 @@ export class FocusChangeHandler {
         this.scrollingController.processRequest(scrollingMessage);
         this.targetPageActionMessageCreator.scrollRequested();
     };
+
+    private getTarget(storeData: TargetPageStoreData): string[] {
+        return (
+            storeData.visualizationStoreData.focusedTarget || this.getCardResultTarget(storeData)
+        );
+    }
+
+    private getCardResultTarget(storeData: TargetPageStoreData): string[] {
+        if (
+            storeData.cardSelectionStoreData.focusedResultUid == null ||
+            isEmpty(storeData.unifiedScanResultStoreData.results)
+        ) {
+            return null;
+        }
+
+        const focusedResult = storeData.unifiedScanResultStoreData.results.find(
+            result => result.uid === storeData.cardSelectionStoreData.focusedResultUid,
+        );
+
+        if (focusedResult == null) {
+            throw 'focused result was not found';
```

Comment:
I think we avoid throwing `string`, it make for weird behavior. Throwing `new Error('focused result was not found')` should be fine.

Reply:
Done!